Library/Homebrew/brew.sh

```diff
@@ -919,6 +919,37 @@ then
   export HOMEBREW_DEVELOPER_COMMAND="1"
 fi
 
+if [[ -n "${HOMEBREW_DEVELOPER}" || -n "${HOMEBREW_DEVELOPER_COMMAND}" ]]
+then
+  # Always run with Sorbet for Homebrew developers or Homebrew developer commands.
+  export HOMEBREW_SORBET_RUNTIME="1"
+fi
+
+# NO_SORBET_RUNTIME_COMMANDS are currently failing with Sorbet for homebrew/core.
+# TODO: fix this and remove this if block.
+if [[ -n "${HOMEBREW_SORBET_RUNTIME}" ]]
+then
+  NO_SORBET_RUNTIME_COMMANDS=(
+    generate-cask-api
+    generate-formula-api
+    release
```

Comment:
@dduugg can any of these be removed now https://github.com/Homebrew/brew/pull/15710 is merged?
@Homebrew/maintainers @Bo98 if you have any other commands you know are failing with Sorbet: please shout so I can add in here before I merge (not today, post 4.1.0).

Reply:
There was stuff in homebrew-core I think. I'll try dig into it tonight.

Reply:
@MikeMcQuaid I believe these entries can be removed, I'm able to run these commands as of https://github.com/Homebrew/brew/pull/15710. (I tried the `generate*` commands with and without `-n`, and I tried `release` but stopped toying with it when I realized it was drafting releases in the `Homebrew` account 🙈).

Reply:
```suggestion
```